Products/zms/ZMSZCatalogOpensearchConnector.py

```diff
@@ -104,7 +149,7 @@ def search_json(self, q, REQUEST, RESPONSE=None):
       index_id = REQUEST.get('opensearch_index_id',root_id)
       username = self.getConfProperty('opensearch.username', 'admin')
       password = self.getConfProperty('opensearch.password', 'admin')
-      verify = bool(self.getConfProperty('opensearch.ssl.verify', ''))
+      verify = bool(self.getConfProperty('opensearch.ssl.verify', False))
```

Comment:
Why? bool('') evaluates to False!?!

Reply:
yes, bool('')==False, luckily ;-)
The ZMS custom property is not typed _a-priori_; so in practice it may be 0, '', 'False', False.
But here the value of variable `verify ` is expected as bool, definitely. Now the reader can recognize, that default=False/bool. The nesting function bool() just ensures the bool-type as long as getConfProperty() provides a dynamic type. Of course both code-variants work. Which is nicer?

Reply:
The initial variant is nicer since conf-properties edited through the ZMI should mostly assumed to be strings, but it does not matter. You cann ignore my comment, I just wanted to question the intention of your change.